### DISCUSSION TITLE
我把网站目录的权限关了，为什么还可以访问

### DISCUSSION THREAD

**Discussion by ly0813** [1Panel-dev/1Panel#6122] [Q&A] [answered]
### 1Panel 版本

v1.10.8-lts

### 问题描述

设置0700权限，还可以访问敏感目录

### 重现步骤

![image](https://github.com/user-attachments/assets/e7409180-de8c-412c-befa-61e164cf3288)
![image](https://github.com/user-attachments/assets/7f88b3be-d8f0-4662-8452-e58b23f23db5)

**Comment by wanghe-fit2cloud** [comment]
麻烦再详细描述一下具体使用场景。
进入 `终端`，执行 `ls -l` 命令也查看一下文件夹具体权限。

**Reply by ly0813** [reply, marked as answer]
![image](https://github.com/user-attachments/assets/db316422-acf5-4bda-8b87-ee5106922c95)  网站目录，我设置了0770文件夹权限，但是还是可以通过域名直接访问这个目录里面的文件夹内容（未登录无痕模式也可以，）/config/  ，/tmp/等敏感的文件夹，网站php环境 7.4.33 ，网站Web 服务器 OpenResty ，

**Reply by zhengkunwang223** [reply]
> ![image](https://private-user-images.githubusercontent.com/53300199/357662817-db316422-acf5-4bda-8b87-ee5106922c95.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM2MjE4NjcsIm5iZiI6MTcyMzYyMTU2NywicGF0aCI6Ii81MzMwMDE5OS8zNTc2NjI4MTctZGIzMTY0MjItYWNmNS00YmRhLThiODctZWU1MTA2OTIyYzk1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE0VDA3NDYwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk4ZGNkMTk1MGFhYzAyOWY3YmI1Y2FhNTM5NTk3Y2NmYzEwYTRiNWRlNGJlNmRhMGFlYTBmNTllNjllMGUyMmMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.mbVvnvUJh_oplDVepKD6BZaYM9o-fR5ih_jXS5joEso) 网站目录，我设置了0770文件夹权限，但是还是可以通过域名直接访问这个目录里面的文件夹内容（未登录无痕模式也可以，）/config/ ，/tmp/等敏感的文件夹，网站php环境 7.4.33 ，网站Web 服务器 OpenResty ，

1Panel 的 OpenResty 的 内部用户是 root 所以无法通过权限的方式来限制  你可以使用  WAF 或者 编写 nginx 配置的情况来实现
